nofos/composer/templates/composer/composer_section_edit.html

```diff
@@ -0,0 +1,86 @@
+{% extends 'base.html' %}
+{% load static tz martortags add_footnote_ids add_captions_to_tables add_classes_to_lists add_classes_to_paragraphs add_classes_to_tables convert_paragraphs_to_hrs nofo_section_name_separator safe_br %}
+
+{% block title %}
+  Preview: {{ document.title }} </title>
+{% endblock %}
+
+{% block css %}
+  <link href="{% static "styles-theme-base.css" %}" type="text/css" media="all" rel="stylesheet">
+  <link href="{% static "theme-orientation-portrait.css" %}" type="text/css" media="all" rel="stylesheet">
+  <link href="{% static "theme-opdiv-cdc.css" %}" type="text/css" media="all" rel="stylesheet">
+  <link href="{% static "theme-opdiv-cdc-blue.css" %}" type="text/css" media="all" rel="stylesheet">
+{% endblock %}
+
+{% block body_class %}composer_document_section composer_document_section--preview{% endblock %}
+
+{% block content %}
+  <div class="grid-row cg-layout">
+    <!-- Full-document preview -->
+    <div class="grid-col-12">
+      <div class="back-link font-body-md margin-bottom-105">
+        <a class="usa-button--icon--before usa-button--arrow_back" href="{% url 'composer:section_view' document.pk section.pk %}">Back to section</a>
+      </div>
+      <h1 class="margin-bottom-1 margin-top-0 font-serif-xl">Configure step
+        <br>
+        {% with section.name|nofo_section_name_separator as section_name %}
+          <span class="font-serif-lg  text-base">{% if section_name.number %}{{ section_name.number }}. {% endif %}{{ section_name.name|safe_br }}</span>
+        {% endwith %}
```

Comment:
I don't think it matters too much if we include "Step 1:" in the name, but also fine to do it this way.